pkg/activator/net/throttler_test.go

```diff
@@ -49,27 +49,28 @@ import (
 
 const defaultMaxConcurrency = 1000
 
+var defaultParams = queue.BreakerParams{
```

Comment:
There is one more of these down in `TestMultipleActivators`

Reply:
Removed.